src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.hpp

```diff
@@ -3,12 +3,54 @@
 
 #pragma once
 
+#include <cstddef>
+#include <memory>
+#include <string>
+#include <unordered_map>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "Domain/Creators/DomainCreator.hpp"
+#include "Domain/Domain.hpp"
+#include "Domain/OptionTags.hpp"
+#include "Domain/Structure/ExcisionSphere.hpp"
 #include "Options/Options.hpp"
 
 namespace CurvedScalarWave::Worldtube {
 namespace OptionTags {
 struct Worldtube {
   static constexpr Options::String help = {"Options for the Worldtube"};
 };
+
+struct ExcisionSphere {
+  using type = std::string;
+  static constexpr Options::String help{
+      "The name of the excision sphere as returned by the domain."};
+  using group = Worldtube;
+};
+
 }  // namespace OptionTags
+
+namespace Tags {
+template <size_t Dim>
+struct WorldtubeRadius : db::SimpleTag {
```

Comment:
The excision sphere essentially _is_ your worldtube, right? So how about you name this `Worldtube` and make it hold the excision sphere. Then you can get the radius, center, etc from it whenever you want. Right now you can just copy the excision sphere from the domain, but we should get #3142 merged so you can make this a reference tag.